tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -60,6 +67,43 @@ describe('amo/components/AddonReviewList', () => {
       return shallow(<AddonReviewListBase {...props} />);
     }
 
+    function HTMLrender({
+      addon = fakeAddon,
+      dispatch = sinon.stub(),
+      errorHandler = new ErrorHandler({
+        id: 'some-error-handler',
+        dispatch: sinon.stub(),
+      }),
+      params = {
+        addonSlug: fakeAddon.slug,
+      },
+      reviews = [fakeReview],
+      ...customProps
+    } = {}) {
+      const loadedReviews = reviews ? getLoadedReviews({ reviews }) : null;
+      const props = {
+        addon: addon && denormalizeAddon(addon),
+        dispatch,
+        errorHandler,
+        i18n: getFakeI18nInst(),
+        location: { query: {} },
+        params,
+        reviewCount: loadedReviews && loadedReviews.length,
+        reviews: loadedReviews,
+        ...customProps,
+      };
+
+      const store = createStore().store;
+
+      return findDOMNode(findRenderedComponentWithType(renderIntoDocument(
```

Comment:
It renders Just blank HTML.
I got this read - https://github.com/airbnb/enzyme/issues/630